examples/zkp_debugger/src/main.rs

```diff
@@ -28,12 +25,12 @@ fn main() {
     let inputs: Vec<ZkpProgramInput> = vec![
         BPField::from(1).into(),
         BPField::from(2).into(),
-        BPField::from(3).into(),
+        BPField::from(4).into(),
     ];
 
-    let proof = runtime.prove(prog, vec![], vec![], inputs);
+    let _proof = runtime.prove(prog, vec![], vec![], inputs);
 
-    proof.unwrap();
+    // proof.unwrap();
 
     loop {
```

Comment:
Maybe not in this pr but the runtime should hold a handle to the server, so that a user can get a hold of it and call something like `runtime.debugger_handle()??` which will keep the process running until Ctrl+c.